client/src/app/components/stardust/Output.tsx

```diff
@@ -258,6 +265,26 @@ class Output extends Component<OutputProps, OutputState> {
         );
     }
 
+    private getExpandedStateInUnlockCondition(unlockCondition?: IUnlockCondition, unlockConditions?: IUnlockCondition[]): boolean {
+        if (!unlockCondition) return false;
+
+        const expirationUnlockCondition = unlockConditions?.find(
+            (cond) => cond.type === UnlockConditionType.Expiration,
+        ) as ExpirationUnlockCondition;
+        const isExpirationConditionPresent = !!expirationUnlockCondition;
+        const isExpirationConditionExpired =
```

Comment:
this logic is not working, this output `0x4bd228a6f865de501c1a4c55990d968de1ea958c74852860f66e2befabd74cdb0000` was spent by the return address unlock condition, not the address unlock condition, because the output was spent after the time expired

Reply:
I feel like we should distinguish between the outputs in the input part, and the outputs in the output part of the tx, 

For the inputs, we need to check if the output was spent before or after the expiration time, it should not be checked against "now", but when the output was spent
For the outputs, i feel like we could for now always expand the address unlock condition and call it a day, at least for now